.github/renovate.json

```diff
@@ -24,7 +24,7 @@
       "matchStrings": [
         "DockerImageName\\.parse\\(\"(?<depName>hashicorp/vault):(?<currentValue>\\d+\\.\\d+.\\d+)\"\\)",
         "image:.*(?<depName>hashicorp/vault):(?<currentValue>\\d+\\.\\d+.\\d+)",
-        "tag:.*\"(?<currentValue>\\d+\\.\\d+.\\d+)\""
+        "tag:.*(?<currentValue>\\d+\\.\\d+.\\d+)"
```

Comment:
Good catch.   The background here was the type inference.   When we use a x.y version, we had to quote the version because otherwise the type would be inferred to be a float.   With x.y.z  the type is unambiguously a string, so the quoting is redundant.

Reply:
We should make the same change to `helm_localstack_overrides.yaml`.  It quoting is redundant.  Then remove the quotes from the RE in the renovate config.

Reply:
^ this one

Reply:
Done